CMakeLists.txt

```diff
@@ -83,6 +85,14 @@ endif()
 if(${PORTFFT_USE_SCLA})
   target_compile_definitions(portfft INTERFACE PORTFFT_USE_SCLA)
 endif()
+if(${PORTFFT_ENABLE_BUFFER_BUILDS})
+  target_compile_definitions(portfft INTERFACE PORTFFT_ENABLE_BUFFER_BUILDS=1)
+else()
+  target_compile_definitions(portfft INTERFACE PORTFFT_ENABLE_BUFFER_BUILDS=0)
+endif()
```

Comment:
Sorry, just another thought: I don't think there is any reason to set the values `0` or `1` to `PORTFFT_ENABLE_BUFFER_BUILDS`, we can just define it or not define it like most of our other macros.

Reply:
done